layers/+tools/shell/funcs.el

```diff
@@ -27,6 +27,15 @@
   (let ((default-directory (projectile-project-root)))
     (call-interactively 'spacemacs/default-pop-shell)))
 
+(defun spacemacs/projectile-run-default-shell ()
+  "Run the default shell at the project root. If projectile
+doesn't support this, use projectile-run-shell instead."
```

Comment:
as for docstring:
- the first line should be a short and complete sentence. So add a new line after the period.
- symbol name should be quoted with <code>&#96;&apos;</code> so  <code>&#96;projectile-run-shell&apos;</code>